.github/workflows/pr_integration_tests.yml

```diff
@@ -95,6 +99,10 @@ jobs:
         run: |
           docker pull vishnunair/docker-redis-cluster:latest
           docker run -d -p 6001:6379 -p 6002:6380 -p 6003:6381 -p 6004:6382 -p 6005:6383 -p 6006:6384 --name redis-cluster vishnunair/docker-redis-cluster
+      - name: Setup Milvus Cluster
+        run: |
+          wget https://github.com/milvus-io/milvus/releases/download/v2.4.20/milvus-standalone-docker-compose.yml -O docker-compose.yml
+          docker compose up -d
```

Comment:
Don't we need to tear down the cluster before ending the job?

Reply:
Nope, we don't for Redis. It'll happen automatically by the action. You can confirm this in the example I linked here: https://github.com/franciscojavierarceo/testing-github-action-sandbox/actions/runs/12602548011/job/35125851535

Reply:
not sure why do we need both one in action and services? do we need at both places? Looks like we are following the same pattern for redis as well.

Reply:
There are some benefits for services but for Milvus, the service isn't necessary. I just followed what I thought was the convention but it's not necessary. I'll update the PR. 👍